cuenca/resources/resources.py

```diff
@@ -18,6 +18,4 @@ def retrieve_uri(uri: str) -> Retrievable:
 
 def retrieve_uris(uris: List[str]) -> List[Retrievable]:
     with ThreadPoolExecutor(max_workers=len(uris)) as executor:
-        return [
-            obj for obj in executor.map(retrieve_uri, [uri for uri in uris])
-        ]
+        return [obj for obj in executor.map(retrieve_uri, uris)]
```

Comment:
@kerycdiaz @alexviquez couldn't this be `list(executor.map(retrieve_uri, uris)`?

Reply:
```python
In [2]: list(range(10)) == [x for x in range(10)]
Out[2]: True
```

Reply:
Cierto Matin